nyxx.interactions/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0
+
+- Initial version, created by Stagehand
```

Comment:
Remember to match version to nyxx (`1.1-dev.1`) and later `1.1`. Also remember to fill this up as you go with stuff

Reply:
Updated version, will add changelog info as I have stuff to add